node-tests/service-worker-builder-test.js

```diff
@@ -127,6 +127,29 @@ describe('Service Worker Builder', () => {
         assert.equal(files[serviceWorkerFilename], expected);
       });
   });
+  it('overides root url when using serviceWorkerScope', () => {
```

Comment:
This is great - can we also test that the root url is used as the scope when the scope is not explicitly passed?

Reply:
Just added tests for this for a custom rootURL and the default `/`.